src/main/java/ladder/Domain/Climber.java

```diff
@@ -0,0 +1,27 @@
+package ladder.Domain;
+
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+public class Climber {
+
+    private Users users;
+
+    private Climber() {
+    }
+
+    public static Climber of() {
+        return new Climber();
+    }
+
+    public LadderMap createLadder(String userNames, int height) {
+        users = Users.of(userNames);
+        return LadderMap.of(users, height, () -> new Random().nextInt(10) >= 5);
```

Comment:
```suggestion
        return LadderMap.of(users, height, isCrossRoad());
    }

    private CrossRoadStrategy isCrossRoad() {
        return () -> new Random().nextInt(LADDER_BOUND) >= LINE_CONDITION;
    }
```
람다표현식을 사용하여 동작을 파라미터화하셨네요! 👍 
메서드 이름을 부여하여 의도를 분명히 드러내는 것은 어떨까요